traefik.toml

```diff
@@ -0,0 +1,34 @@
+#debug = true
+
+defaultEntryPoints = ["http", "https"]	# defaultEntryPoints must be at the top because it should not be in any table below
+
+[web]
+  # Port for the status page
+  address = ":8080"
+  #[web.auth.basic]
+  #  users = ["traefik:generate-with-htpasswd"]
+
+# Entrypoints, http and https
+[entryPoints]
+  [entryPoints.http]					# http should be redirected to https
+    address = ":80"
+    [entryPoints.http.redirect]
+      entryPoint = "https"				# https is the default
+  [entryPoints.https]
+    address = ":443"
+    [entryPoints.https.tls]
+
+# Enable ACME (Let's Encrypt): automatic SSL
+[acme]
+  # caServer = "https://acme-staging.api.letsencrypt.org/directory"
+  email = "me@domain.tld"
+  storage = "/etc/traefik/acme/acme.json" 				# or "traefik/acme/account" if using KV store
+  entryPoint = "https"
+  onDemand = false
+  OnHostRule = true
+
+[docker]
+  endpoint = "unix:///var/run/docker.sock"
+  domain = "toolz.pm"
```

Comment:
Shouldn't this be `domain.tld` and not `toolz.pm`?

Reply:
https://github.com/hardware/mailserver/blob/master/traefik.sample.toml#L22 ;)

Reply:
Cool. Thanks.